### ISSUE TITLE
Calculate potential impact of IP-profiling feature

### ISSUE THREAD

**Issue by rfk** [mozilla/fxa-auth-server#1517]
Before we invest too much more into the "ip profiling" feature to reduce sign-in confirmation, let's use server-side log data to see what its potential impact could have been.  Specifically:
- How many successful login attempts came from IPs we'd previously seen for that user?

I've asked for an anonymized dump of the relevant server data here:

  https://bugzilla.mozilla.org/show_bug.cgi?id=1311891

Once we have that, we can import into redshift and run some analysis.

**Comment by rfk** [comment]
I'm moving this into a MoCo-confidential bugzilla bug so we can discuss raw metrics:

  https://bugzilla.mozilla.org/show_bug.cgi?id=1313275

With some preliminary analysis here:

 https://bugzilla.mozilla.org/show_bug.cgi?id=1313275#c1

TL;DR: it looks surprisingly impactful!

**Comment by rfk** [comment]
I think we've got enough metrics to give a 👍 to this feature, closing out the bug.
